src/test/java/it/pagopa/pn/delivery/middleware/notificationdao/NotificationMetadataEntityDaoDynamoTest.java

```diff
@@ -54,7 +54,7 @@ void InsertMetadataEntitySuccess() throws PnIdConflictException {
                 .recipientId( "RecipientId" )
                 .build();
 
-        InputSearchNotificationDto searchDto = new InputSearchNotificationDto.Builder()
+        InputSearchNotificationDto searchDto = new InputSearchNotificationDto().toBuilder()
```

Comment:
Come mai è diventato toBuilder e l'oggetto non è un builder?

Reply:
il codeSmell richiedeva di limitare il numero di parametri del costruttore, ho quindi creato un costruttore senza argomenti e modificato il codice utilizzando il toBuilder().build invece dell' Builder().build utilizzato in precedenza